mlte/backend/api/auth/authentication.py

```diff
@@ -0,0 +1,26 @@
+"""
+mlte/backend/api/auth/authentication.py
+
+Authentication handling.
+"""
+
+from mlte.store.user.store import UserStoreSession
+from mlte.user import passwords
+
+
+def authenticate_user(
+    username: str, password: str, user_store_session: UserStoreSession
+) -> bool:
+    """Validates the credentials."""
+    user = None
+    try:
+        user = user_store_session.read_user(username)
+    except Exception:
+        # Assume any exception means we couldn't load user it.
+        # print(f"Error reading user: {ex}")
```

Comment:
Did you want to leave this print and the one a couple lines down in as comments or remove/uncomment?

Reply:
I wanted to leave them commented out intentionally, for now at least. They are useful when debugging, which we may need for the authorization part, but they leak some potentially sensitive authentication info if always enabled.